homeassistant/components/rympro/sensor.py

```diff
@@ -17,6 +20,37 @@
 from .coordinator import RymProDataUpdateCoordinator
 
 
+@dataclass
+class RymProSensorDescription(SensorEntityDescription):
+    """Class describing rympro sensor entities."""
+
+    sensor: MeterSensor | None = None
```

Comment:
You should use a Mixin class to ensure the key is required.

```suggestion
@dataclass
class RymProSensorRequiredKeysMixin:
    """Mixin for require keys."""

    sensor: MeterSensor


@dataclass
class RymProSensorDescription(SensorEntityDescription, RymProSensorRequiredKeysMixin):
    """Class describing rympro sensor entities."""

```